toolkit/ar/src/main/java/com/arcgismaps/toolkit/ar/WorldScaleSceneView.kt

```diff
@@ -73,6 +74,68 @@ import com.arcgismaps.toolkit.geoviewcompose.SceneView
 import com.arcgismaps.toolkit.geoviewcompose.SceneViewDefaults
 import java.time.Instant
 
+/**
+ * A scene view that provides an augmented reality world scale experience.
+ *
+ * Note: You must follow [Google's user privacy requirements for ARCore](https://developers.google.com/ar/develop/privacy-requirements)
+ * when using this WorldScaleSceneView in your application.
+ *
+ * @param arcGISScene the [ArcGISScene] to be rendered by this WorldScaleSceneView
+ * @param modifier the modifier to be applied to the WorldScaleSceneView
+ * @param worldScaleTrackingMode the type of tracking configuration used by the WorldScaleSceneView.
+ * This determines how the position and orientation of the device is obtained and synchronized with
+ * the scene view's camera.
+ * @param clippingDistance the clipping distance in meters around the scene view's camera. A null
+ * means that no data will be clipped.
+ * @param onInitializationStatusChanged a callback that is invoked when the initialization status
+ * of this WorldScaleSceneView changes.
+ * @param onViewpointChangedForCenterAndScale lambda invoked when the viewpoint changes, passing a
+ * viewpoint type of [ViewpointType.CenterAndScale]
+ * @param onViewpointChangedForBoundingGeometry lambda invoked when the viewpoint changes, passing a
+ * viewpoint type of [ViewpointType.BoundingGeometry]
+ * @param graphicsOverlays graphics overlays used by this WorldScaleSceneView
```

Comment:
In keeping with parameters below:
```suggestion
 * @param graphicsOverlays graphics overlays used by the WorldScaleSceneView
```